README.md

```diff
@@ -350,13 +351,13 @@ It is possible to develop ORY Kratos on Windows, but please be aware that all gu
 
 #### Install from source
 
-```shell script
+<pre type="make/command">
 make install
-```
+</pre>
```

Comment:
Here I describe that this code block contains a make command. Text-Runner looks inside the Makefile and makes sure the documented targets exist there.